packages/fxa-content-server/app/tests/spec/views/support.js

```diff
@@ -210,6 +274,8 @@ describe('views/support', function() {
             supportTicket
           );
           assert.isTrue(view.navigateToSubscriptionsManagement.calledOnce);
+          assert.isTrue(view.logFlowEvent.calledOnce);
+          assert.isTrue(view.logFlowEvent.calledWith('success', 'support'));
```

Comment:
Fwiw, I sometimes find these `isTrue` assertions unhelpful for debugging test failures because they just give a message like "expected false to be true". Where possible I think it produces more useful error messages if you can assert the actual value, so `assert.equal(view.logFlowEvent.callCount, 1)` in this case. Or in some places we use the assertions module from sinon itself, and those also produce helpful failures.

Not saying you have to do that here, just wanted to point it out in case it wasn't something you'd considered.